cognite/neat/_data_model/_analysis.py

```diff
@@ -343,3 +343,63 @@ def connection_end_node_types(self) -> dict[tuple[ViewReference, str], ViewRefer
                         connection_end_node_types[(view_ref, prop_ref)] = property_.source
 
         return connection_end_node_types
+
+    @cached_property
+    def container_to_views(self) -> dict[str, set[str]]:
+        """Get a mapping from container reference strings to the views that use them.
+
+        Returns:
+            Dictionary mapping container string (e.g., 'space:containerId') to set of view strings
+        """
+        container_to_views: dict[str, set[str]] = {}
+
+        for view_ref, view in self.merged.views.items():
+            if not view.properties:
+                continue
+            for property_ in view.properties.values():
+                if isinstance(property_, ViewCorePropertyRequest):
+                    container_str = str(property_.container)
+                    if container_str not in container_to_views:
+                        container_to_views[container_str] = set()
+                    container_to_views[container_str].add(str(view_ref))
+
+        return container_to_views
+
+    @cached_property
+    def view_to_containers(self) -> dict[str, set[ContainerReference]]:
```

Comment:
```suggestion
    def views_to_containers(self) -> dict[ViewReference, set[ContainerReference]]:
```

view -> views
str -> ViewReference